### DISCUSSION TITLE
trigger syntax

### DISCUSSION THREAD

**Discussion by icegood** [dbeaver/dbeaver#13985] [Q&A]
#### System information: 
- Operating system (distribution) and version
Linux ice-ubuntu 5.11.0-18-generic #19-Ubuntu SMP Fri May 7 14:22:03 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
- DBeaver version
- Version 21.1.0.202106012023
- Additional extensions

#### Connection specification:
- Database name and version sqlite
- Driver name org.xerial:sqlite-jdbc:RELEASE
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
trigger view is not SQL formatted
see picture
https://drive.google.com/file/d/1X70PgxFr-Gi2aYDnFf3nNYaW1qutJamb/view?usp=sharing

**Comment by uslss** [comment]
Hi @icegood ,
unfortunately, cannnot reproduce the issue (at least against dbeaver sample sqlite database, picture attached). Probably, you could provide ddl that will help to reproduce an issue? 
![image](https://user-images.githubusercontent.com/31996417/122016735-fb1c7980-cdc9-11eb-9e1a-9fcf6676301a.png)

**Comment by icegood** [comment]
Sure, 

CREATE TRIGGER raw_contacts_deleted    BEFORE DELETE ON raw_contacts BEGIN    DELETE FROM data     WHERE raw_contact_id=OLD._id;   DELETE FROM agg_exceptions     WHERE raw_contact_id1=OLD._id        OR raw_contact_id2=OLD._id;   DELETE FROM visible_contacts     WHERE _id=OLD.contact_id       AND (SELECT COUNT(*) FROM raw_contacts            WHERE contact_id=OLD.contact_id           )=1;   DELETE FROM default_directory     WHERE _id=OLD.contact_id       AND (SELECT COUNT(*) FROM raw_contacts            WHERE contact_id=OLD.contact_id           )=1;   DELETE FROM contacts     WHERE _id=OLD.contact_id       AND (SELECT COUNT(*) FROM raw_contacts            WHERE contact_id=OLD.contact_id           )=1; END;

**Comment by uslss** [comment]
thank you for the report

The trigger source is shown as it was when the trigger was created. If the SQL query was formatted before execution it will be shown the same way at the source tab.
![image](https://user-images.githubusercontent.com/31996417/123618599-07182a80-d811-11eb-94f2-7e129fe10de5.png)

**Comment by LonwoLonwo** [comment]
Hello @icegood 

We are reading objects DDL from sqlite_master. And sqlite_master stores it in the format in which the statement was performed. 
This is a standard situation for many databases. So what can we help you with?
